src/routes/analytics/AnalyticsCustomPage.tsx

```diff
@@ -1,18 +1,43 @@
-import { Box } from "@mantine/core"
-import { useLocation } from "wouter"
+import { useAtom } from "jotai"
+import { Link, useLocation } from "wouter"
+import { Button, Container, Flex } from "@mantine/core"
 import { CollectionBrowser } from "@metabase/embedding-sdk-react"
 
-import "./analytics-custom-page.css"
+import { NewQuestionMenu } from "../../components/NewQuestionMenu"
 import { RemountOnSiteChange } from "../../components/RemountOnSiteChange"
 
+import { siteAtom } from "../../store/site"
+import { createDashboardIdAtom } from "../../store/create"
+
+import { SANDBOXED_CUSTOM_ANALYTICS_COLLECTIONS } from "../../constants/collections"
+
+import "./analytics-custom-page.css"
+
 export function AnalyticsCustomPage() {
   const [, navigate] = useLocation()
+  const [, setDashboardId] = useAtom(createDashboardIdAtom)
+
+  const [site] = useAtom(siteAtom)
+  const collectionId = SANDBOXED_CUSTOM_ANALYTICS_COLLECTIONS[site]
 
   return (
-    <Box p={20}>
+    <Container w="100%" p={20}>
+      <Flex justify="flex-end" gap="xs" pb="xs">
+        <NewQuestionMenu position="bottom-end">
+          <Button>New Question</Button>
+        </NewQuestionMenu>
+
+        <Link
+          href="/analytics/new/dashboard"
+          onClick={() => setDashboardId(null)}
+        >
+          <Button>New Dashboard</Button>
+        </Link>
+      </Flex>
```

Comment:
Add buttons to create new question and new dashboard.

![CleanShot 2567-09-04 at 02 13 56@2x](https://github.com/user-attachments/assets/8ad73df4-3b74-43cf-9834-895a6c925a85)